posts/workshop-vignette/index.qmd

```diff
@@ -0,0 +1,349 @@
+---
+title: "workshop vignette"
+output: 
+  pdf_document:
+    latex_engine: xelatex
+date: "2023-01-12"
+self_contained: TRUE
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = F)
+```
+<br>
+
+This vignette summarises the findings from the _100 days and 100 lines of code_ workshop, hosted in December 2022 by [Epiverse-TRACE](https://data.org/news/epiverse-trace-a-values-based-approach-to-open-source-ecosystems/).
+
+
+## What should the first 100 lines of code written during an epidemic look like?
+
+To answer this question, we invited 40 experts, including academics, field epidemiologists, and software engineers, to take part in a 3-day workshop, where they discussed the current challenges, and potential solutions, in data analytic pipelines used to analyse epidemic data.
+
+### What R packages and tools are available to use during an epidemic?  
+To investigate this in a similar setting to what an outbreak response team would experience, workshop participants were divided into groups, and asked to develop a plausible epidemic scenario, that included:
+
+* A situation report, describing the characteristics of the epidemic
+
+* A linelist of cases and contact tracing data, by modifying provided datasets containing simulated data
+
+* A set of questions to address during the analytic process
+
+Groups then exchanged epidemic scenarios and analysed the provided data to answer the questions indicated the previous group, as if they were a response team working to solve an outbreak.
+Details about each of these outbreak scenarios and the analytic pipelines developed by the groups are summarised in this vignette.
+
+### Simulating epidemic data
+
+Before the workshop, a fictitious dataset was created, which consisted of a linelist and contact tracing information.
+
+To generate linelist data, the package [`bpmodels`](https://github.com/epiverse-trace/bpmodels) was used to generate a branching process network. Cases were then transformed from the model output to a linelist format. To add plausible hospitalisations and deaths, delay distributions for SARS-CoV were extracted from [`epiparameter`](https://github.com/epiverse-trace/epiparameter).
+
+To create the contact tracing database, a random number of contacts was generated for each of the cases included in the linelist. These contacts were then assigned a category of _became case_, _under follow up_ or _lost to follow up_, at random.
+
+* Through this workshop, we identified the need for a tool to simulate outbreak data in a linelist format, to test analysis methods and other packages while having control over the characteristics of the test data. For this purpose, an R package is currently in progress, see [simulist](https://github.com/epiverse-trace/pitcher/issues/7).
+
+## Scenario 1: Novel respiratory disease in The Gambia
+
+<br>
+```{r, echo=FALSE, out.width="100%"}
+knitr::include_graphics('posts/workshop-vignette/Scenario1smq.png')
```

Comment:
```suggestion
knitr::include_graphics('Scenario1smq.png')
```

Paths in Rmarkdown (`.Rmd`) and Quarto (`.qmd`) documents are relative to the document where they are called. So here, relative to `index.qmd`. Since your images are in the same folder as `index.qmd`, you should call them directly.